mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -126,13 +126,15 @@ private[python] class PythonMLLibAPI extends Serializable {
       k: Int,
       maxIterations: Int,
       minDivisibleClusterSize: Double,
-      seed: Long): BisectingKMeansModel = {
-    new BisectingKMeans()
+      seed: java.lang.Long): BisectingKMeansModel = {
```

Comment:
To complete the thought here: I discovered that most of the Pyspark API methods accepted a `java.lang.Long`, in order to only optionally accept a seed. A few did not however. I also made those consistent, so that the Pyspark default of seed=None results in a null to this argument, which results in a default seed in the JVM, which means a random seed.